core-it-support/core-it-plugins/maven-it-plugin-site/pom.xml

```diff
@@ -72,6 +72,12 @@ under the License.
       <groupId>org.apache.maven.reporting</groupId>
       <artifactId>maven-reporting-exec</artifactId>
       <version>1.6.0</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.maven</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
This has been already done as part of https://github.com/apache/maven-integration-testing/pull/223, but then I think it got accidentally reverted when merging some other changes.